frontend/kesaseteli/employer/src/hooks/employments/useWatchEmployeeDisplayName.ts

```diff
@@ -0,0 +1,14 @@
+import useApplicationFormField from 'kesaseteli/employer/hooks/application/useApplicationFormField';
+import { useTranslation } from 'next-i18next';
+import { isEmpty } from 'shared/utils/string.utils';
+
+const useWatchEmployeeDisplayName = (index: number): string => {
+  const { t } = useTranslation();
+  const defaultHeading = `${t(`common:application.step2.employment`)} #${
+    index + 1
+  }`;
+  const { watch: watchEmployeeName } = useApplicationFormField<string>(`summer_vouchers.${index}.employee_name`);
+  const employeeName = watchEmployeeName();
+  return !employeeName || isEmpty(employeeName) ? defaultHeading : employeeName;
```

Comment:
`isEmpty` can handle that, can't it?
```suggestion
  return isEmpty(employeeName) ? defaultHeading : employeeName;
```

Reply:
yes it does but typescript does not catch it so the first term is for typescript only :(
afaik in the future versions of typescript this will be fixed https://devblogs.microsoft.com/typescript/announcing-typescript-4-4/#cfa-aliased-conditions
Actually 4.4 is now released but we are using version 4.3.4 I could try to update and remove the first term.

Reply:
seem to work